actions.go

```diff
@@ -276,7 +276,7 @@ func (c *Action) Fatalf(msg string, args ...interface{}) {
 // arguments.
 func (c *Action) Infof(msg string, args ...interface{}) {
 	// ::info <c.fields>::<msg, args>
-	fmt.Fprintln(c.w, msg, args)
+	fmt.Fprintf(c.w, msg, args...)
```

Comment:
Other conptempory functions for writing logs internally append a new line to the message. The TS toolkit also [appends a newline at the end](https://github.com/actions/toolkit/blob/383ec9fb0389e0473078971d5423323e79a575ff/packages/core/src/core.ts#L168-L170).

Reply:
Hmm - should we revert and update the godoc?

@jlordiales what's the use case for _not_ including EOF?

Reply:
no reason, just didn't realize that a newline was added automatically. 
Personally I'd expect that to work the same as if I was doing `fmt.Printf(...` where I would include the new line explicitly if I wanted it. But happy to add it on a separate PR as well

Reply:
@sethvargo The proposed fix addresses a different issue (placeholders in messages not getting formatted with specified args). The newline thing appears to be an oversight.

Reply:
Hmm okay. I was hesitant that this might break things. Here's what I propose: revert this and update the godoc to make it clear that a newline char is added (and that is different from printf). Thoughts @ashutoshgngwr @jlordiales ?

Reply:
@jlordiales @sethvargo Personally, I think it's good either way as long as it is reflected in the documentation, but adding a newline makes it consistent with the rest of the API.